flow-decomposition/src/main/java/com/powsybl/flow_decomposition/XnecProvider.java

```diff
@@ -6,19 +6,39 @@
  */
 package com.powsybl.flow_decomposition;
 
+import com.powsybl.contingency.ContingenciesProvider;
 import com.powsybl.iidm.network.Branch;
 import com.powsybl.iidm.network.Network;
+import org.checkerframework.checker.nullness.qual.NonNull;
```

Comment:
Perhaps prefer javax.annotation.Nonnull ? Seems nothing perfect available for it currently https://stackoverflow.com/questions/4963300/which-notnull-java-annotation-should-i-use